src/utils.js

```diff
@@ -0,0 +1,53 @@
+/*
+  Utilities functions and polyfills
```

Comment:
Useless file

Reply:
those exported functions are critical ...

Reply:
Check my pull request - they are not needed